src/main/webapp/assets/modules/get-promoted/gp-result.mod.js

```diff
@@ -0,0 +1,36 @@
+techlooper
+  .directive("getPromotedResults", function ($http, $translate, utils) {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/get-promoted/gp-result.tem.html",
+      link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.